Code/Framework/AzToolsFramework/Tests/UI/EntityOutlinerTests.cpp

```diff
@@ -72,13 +67,8 @@ namespace UnitTest
             AZ::Entity* entity = nullptr;
             AZ::ComponentApplicationBus::BroadcastResult(entity, &AZ::ComponentApplicationRequests::FindEntity, entityId);
 
-            entity->Deactivate();
+            AddRequiredEditorComponents(entity);
```

Comment:
Is this needed? I think m_prefabPublicInterface->CreateEntity() already adds the default components now that you have connected the editor bus.

Reply:
nice catch!, I will remove it.